### PR TITLE
chore: organize sdk library & sample code library

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

* chore: organize sdk library & sample code library

## Linear Issue Reference

* [organize sdk library & sample code library (LOG-317)](https://linear.app/silverhand/issue/LOG-317/organize-sdk-library-and-sample-code-library)

## Testing

None

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -35,16 +35,14 @@ dependencies {
...
-    implementation "com.google.android.material:material:$rootProject.materialVersion"
-    implementation "androidx.constraintlayout:constraintlayout:$rootProject.constraintlayoutVersion"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$rootProject.coroutinesVersion"
     implementation "org.bitbucket.b_c:jose4j:$rootProject.jose4jVersion"
 
     // app specified
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.navigation:navigation-fragment-ktx:2.3.5'
+    implementation "com.google.android.material:material:1.4.0"
+    implementation "androidx.constraintlayout:constraintlayout:2.1.2"
```
Comment (IceHe) [inline android/app/build.gradle:45]: better in lexicographical order?

Comment (xiaoyijun) [reply android/app/build.gradle:45]: Not like `imports`, we organized 3rd libs according to there responsibilities

Comment (IceHe) [reply android/app/build.gradle:45]: ok

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**